src/sql/workbench/browser/modal/modal.ts

```diff
@@ -83,52 +83,52 @@ const tabbableElementsQuerySelector = 'a[href], area[href], input:not([disabled]
 
 export abstract class Modal extends Disposable implements IThemable {
 	protected _useDefaultMessageBoxLocation: boolean = true;
-	protected _messageElement: HTMLElement;
+	protected _messageElement?: HTMLElement;
 	protected _modalOptions: IModalOptions;
 	protected readonly disposableStore = this._register(new DisposableStore());
-	private _detailsButtonContainer: HTMLElement;
-	private _messageIcon: HTMLElement;
-	private _messageSeverity: HTMLElement;
-	private _messageSummary: HTMLElement;
-	private _messageBody: HTMLElement;
-	private _messageDetail: HTMLElement;
-	private _toggleMessageDetailButton: Button;
-	private _copyMessageButton: Button;
-	private _closeMessageButton: Button;
-	private _messageSummaryText: string;
-	private _messageDetailText: string;
-
-	private _spinnerElement: HTMLElement;
-	private _firstTabbableElement: HTMLElement; // The first element in the dialog the user could tab to
-	private _lastTabbableElement: HTMLElement; // The last element in the dialog the user could tab to
-	private _focusedElementBeforeOpen: HTMLElement;
+	private _detailsButtonContainer?: HTMLElement;
```

Comment:
Not the best approach since this just hides the problem rather than ensuring code correctness, but it would take a while to actually fix the code.